distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -398,22 +196,27 @@ public void beforeExam() throws Exception {
     }
 
     @Test
-    public void testStartUpWithExistingManagedServiceConfigurationFile() throws Exception {
+    public void aRunFirstTestStartUpWithExistingManagedServiceConfigurationFile() throws Exception {
         managedServiceStartupConfig.assertConfigurationPropertiesSet(configAdmin);
         managedServiceStartupConfig.assertFileMovedToProcessedDirectory();
     }
 
     @Test
-    public void testCreateNewManagedServiceConfigurationFile() throws Exception {
+    public void bRunFirstTestCreateNewManagedServiceConfigurationFile() throws Exception {
```

Comment:
Should we start all those methods with `aRunFirst`? That would reduce the risk of someone naming a test method something like `bestTestEver()` and have it run before `cRunFirst`. Thoughts?